mcp-server/bin/run.sh

```
@@ -0,0 +1,17 @@
+# `command`/`args` declared in ../../.claude-plugin/plugin.json.
+#
+# Claude Code spawns plugin-declared MCP server commands with a minimal PATH
+# that does not include version-manager shims (mise, nvm, volta, asdf, ...),
+# so a bare `command: "node"` fails with "Executable not found in $PATH" on
+# any machine where node is only installed through one of those (not through
+# an OS-default location like /usr/local/bin or /opt/homebrew/bin). build.sh
+# records the exact node binary it resolved (and used to build this plugin)
+# into .node-path; invoke that directly by absolute path so this launcher
+# does not depend on PATH at all. Falls back to a bare `node` PATH lookup for
+# a fresh checkout that hasn't been built yet.
+DIR="$(CDPATH= cd -- "$(dirname -- "$0")" && pwd)"
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)